data/sql/updates/pending_db_world/rev_1665246218923522400.sql

```diff
@@ -0,0 +1,3 @@
+--
+UPDATE `creature_template_addon` SET `auras` = '25801' WHERE (`entry` = 15229);
+UPDATE `creature_addon` SET `auras` = '25801' WHERE `guid` BETWEEN 87901 AND 87906;
```

Comment:
Assuming that the spawned creature guid is linked to the entry in creature_template you do not need to update creature_addon

Reply:
It didn't work if I only updated creature_template_addon, I tried.

Reply:
then there is some otherlying problem, as creature_template_anything should be the base for all creatures and creature_anything acts as the override.

Reply:
I'll try to see what's going on

Reply:
A question, though, wouldn't an empty auras field be an override too? What do I place there so that there is no auras?

Reply:
Never thought of that, I suppose that's never the case?